sdk/lib/rtl/threadpool.c

```diff
@@ -837,6 +837,8 @@ static void WINAPI timer_queue_thread_proc(LPVOID p)
 #endif
 }
 
+#ifndef __REACTOS__
```

Comment:
> revert

Did "you" have a look at the 2*2 upstream commits since the import, in case they would help/fix?